### DISCUSSION TITLE
Clarify Database Requirements

### DISCUSSION THREAD

**Discussion by pxlfrk** [umami-software/umami#2021] [Q&A]
Hi,

can you clarify the database requirements and list the supported postgres-Versions in the setup docs?
I can`t find this information anywhere..

This should be useful for users who want to keep their database-versions up2date on their systems..

Thanks in advance! :)

**Comment by pxlfrk** [comment]
@mikecao would you mind having a look into this? Thanks in advance! :)

**Comment by mikecao** [comment]
Umami supports [MySQL](https://www.mysql.com/) (minimum v5.7) and [Postgresql](https://www.postgresql.org/) (minimum v12.14) databases.

From https://umami.is/docs/install

**Reply by pxlfrk** [reply]
Thanks for your reply! My question was focused more on details, so also the maximum least database version, e.g. is umami compatible to postgres 15, mmysql/mariadb 10.11/11 and ongoing.

For myself & my homelab, i try to keep my db stack up2date and try to use the latest LTS versions..
